discord/command.go

```diff
@@ -53,6 +59,39 @@ type Command struct {
 	Version Snowflake `json:"version,omitempty"`
 }
 
+type CommandLocales struct {
```

Comment:
I think it would be best if you implement this as

```go
type CommandLocales map[Language]string

// Language is a type that describes a language code such as "de" or "en-US".
type Language string

const (
	EnglishUS Language = "en-US"
	EnglishUK Language = "en-UK"
	Spanish   Language = "es-ES"
	// ...
)
```

This will also make `,omitempty` actually work.